contracts/epochs/FlowClusterQC.cdc

```diff
@@ -96,13 +96,13 @@ pub contract FlowClusterQC {
         }
 
         /// Returns the number of nodes in the cluster
-        pub fun size(): UInt16 {
+        access(all) fun size(): UInt16 {
             return UInt16(self.nodeWeights.length) 
         }
 
         /// Returns the minimum sum of vote weight required in order to be able to generate a
         /// valid quorum certificate for this cluster.
-        pub view fun voteThreshold(): UInt64 {
+        access(all) view fun voteThreshold(): UInt64 {
             if self.totalWeight == 0 as UInt64 {
                 return 0 as UInt64
             }
```

Comment:
is it stable cadence problem ? 

can't be ?  
```
if self.totalWeight == 0 {
                 return 0 
}
 ```

Reply:
Yeah those are just holdovers from older versions of Cadence when we didn't have type inference. I'll change them. Thanks for pointing it out!